src/test/java/core/basesyntax/service/impl/CsvDataReaderTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+import org.junit.Test;
+
+public class CsvDataReaderTest {
+    private static final String INPUT_VALID_FILE =
+                                "src/test/test-resources/input-files/TestValidInputFile.txt";
+    private static final String INPUT_EMPTY_FILE =
+                                "src/test/test-resources/input-files/TestEmptyFile.txt";
+    private static final String INPUT_NONEXISTENT_FILE =
+                                "src/test/test-resources/input-files/doesNotExist.txt";
+    private static final String INPUT_RANDOM_LINES_FILE =
+                                "src/test/test-resources/input-files/TestRandomLines.txt";
+    private static final CsvDataReader csvDataReader = new CsvDataReader();
+
+    @Test
+    public void readFileLines_allValidConditions_Ok() {
+        List<String> actual = csvDataReader.readFileLines(INPUT_VALID_FILE);
+        List<String> expected = List.of("type,fruit,quantity", "b,banana,20");
+        assertEquals(actual, expected);

```

Comment:
```suggestion
        assertIterableEquals(expected, actual);
```

Reply:
1. first parameter is expected, second is actual.
2. For testing `Iterable` you should use `assertIterableEquals `